components/data-feed-service/dao/destination.go

```diff
@@ -27,7 +28,7 @@ func addToDBDestination(inDestination *datafeed.AddDestinationRequest) *Destinat
 
 func updateToDBDestination(inDestination *datafeed.UpdateDestinationRequest) *Destination {
 	newDestination := Destination{}
-	newDestination.ID, _ = strconv.ParseInt(inDestination.Id, 10 , 64)
+	newDestination.ID, _ = strconv.ParseInt(inDestination.Id, 10, 64)
```

Comment:
why no err check here? is it bc we're assuming that if it's a bad value we would then end up with no update and just return the not found error? would there be any value to checking for the parseint error and returning an err if that doesn't work?  is inDestination.Id a user provided value?

Reply:
That's existing code, it's only a format change from make fmt. 

The inDestination.Id will be provided by the UI thus an existing valid id or any hand crafted external call to the API, this is likely the only case where it could be a non-integer string. The service now app will also call the API as it can configure the datafeed from the service now UI  but that again will be a valid id.  If it's not an integer I just decided to let the database error cascade back, rather than adding additional error handling